### PR TITLE
docs(discovery.md): Improve the description of service discovery

### PR DESCRIPTION
**Pull request by baiyutang**
#### What type of PR is this?

docs: discovery.md
#### What this PR does / why we need it (en: English/zh: Chinese):

en:  Improve the description of service discovery
zh: 完善服务发现的讲述

### CODE REVIEW

File: content/zh/docs/kitex/Tutorials/service-governance/discovery.md [hunk 1]
```diff
@@ -5,14 +5,17 @@ weight: 1
 description: >
 ---
 
-目前在 Kitex 的开源版本中，暂时只提供了一种服务发现的扩展支持 : DNS Resolver, 适合使用 DNS 作为服务发现的场景， 常见的用于 [Kubernetes](https://kubernetes.io/) 集群。
-扩展库：[扩展仓库](https://github.com/kitex-contrib)
+Kitex 已经通过社区开发者的支持，完成了 ETCD、ZooKeeper、Eureka、Consul、Nacos、Polaris 多种服务发现模式，当然也支持 DNS 解析以及 Static IP 直连访问模式，建立起了强大且完备的社区生态，供用户按需灵活选用。
+
+比如 DNS Resolver, 适合使用 DNS 作为服务发现的场景， 常见的用于 [Kubernetes](https://kubernetes.io/) 集群。
+
+更多服务发现组件参看扩展仓库：[registry-etcd](https://github.com/kitex-contrib/registry-etcd)、[registry-nacos](https://github.com/kitex-contrib/registry-nacos)、[registry-zookeeper](https://github.com/kitex-contrib/registry-zookeeper)、[registry-polaris](https://github.com/kitex-contrib/registry-polaris)、[registry-eureka](https://github.com/kitex-contrib/registry-eureka)、[registry-consul](https://github.com/kitex-contrib/registry-consul) 。
```
Comment (li-jin-gou) [inline content/zh/docs/kitex/Tutorials/service-governance/discovery.md:12]: polaris 的仓库地址需要换一下啦

Comment (baiyutang) [reply content/zh/docs/kitex/Tutorials/service-governance/discovery.md]: > polaris 的仓库地址需要换一下啦

DONE

File: content/zh/docs/kitex/Tutorials/service-governance/discovery.md [hunk 2]
```diff
@@ -5,14 +5,17 @@ weight: 1
 description: >
 ---
 
-目前在 Kitex 的开源版本中，暂时只提供了一种服务发现的扩展支持 : DNS Resolver, 适合使用 DNS 作为服务发现的场景， 常见的用于 [Kubernetes](https://kubernetes.io/) 集群。
-扩展库：[扩展仓库](https://github.com/kitex-contrib)
+Kitex 已经通过社区开发者的支持，完成了 ETCD、ZooKeeper、Eureka、Consul、Nacos、Polaris 多种服务发现模式，当然也支持 DNS 解析以及 Static IP 直连访问模式，建立起了强大且完备的社区生态，供用户按需灵活选用。
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/service-governance/discovery.md:8]: 还少了个 ServiceComb

File: content/zh/docs/kitex/Tutorials/service-governance/discovery.md [hunk 3]
```diff
@@ -21,10 +24,11 @@ import (
 
 func main() {
     ...
+    // 以 DNS Resolver 为例
```
Comment (GuangmingLuo) [inline content/zh/docs/kitex/Tutorials/service-governance/discovery.md:27]: 这句话补充到代码块前面吧

**Comment by GuangmingLuo** [comment]
PR title 最好使用英文表述

File: content/zh/docs/kitex/Tutorials/service-governance/discovery.md (hunk 2 above)
Comment (baiyutang) [reply content/zh/docs/kitex/Tutorials/service-governance/discovery.md:8]: done

File: content/zh/docs/kitex/Tutorials/service-governance/discovery.md (hunk 3 above)
Comment (baiyutang) [reply content/zh/docs/kitex/Tutorials/service-governance/discovery.md]: done

**Review by GuangmingLuo — APPROVED** [review]

**Pull request merged**